common/test_local_filestore.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for local_filestore.py."""
```

Comment:
Are you planning to add tests for cp/rm/ls as well?

Reply:
Good suggestions, any concrete plan for writing? Since they are so common, don't know how to design that.
I plan to write a comparison test between local and cloud later after we land the local support.

Reply:
I'd just add at least one for each that tests its functionality (e.g., ls() returns the string what we expect, etc). The fake fs helps.

Reply:
wdyt? @jonathanmetzman Do we also need to add similar cases in `test_gsutil`?
I prefer write a comparison test between `gsutil` and `local_filestore` in `test_filestore_utils`. But that is not for this PR.

Reply:
Writing the tests that verifies the functionality and/or tests that compare functionality between the two impls seems like a good idea.
Note that because we are calling out to real binaries fakefs wont help you'll need to do use tmp_path which is a real path that is temporary.